### PR TITLE
Add the ability to set public access to uploaded files

### LINKED ISSUE #31: Set ACL for uploaded files
We need to be able to set the ACL (set read only for all) for the files (generated metainformation) uploaded to S3. Seems like this option can be set over arguments or / and environment variable.

### PR DESCRIPTION
**Pull request by LeonidVas**
Added the ability to set public access to files uploaded to S3.

Closes #31

### CODE REVIEW

**Review by Totktonada — APPROVED** [review]
Looks meaningful.

File: README.md [hunk 1]
```diff
@@ -49,6 +50,8 @@ So you may skip them in command line invocation in case you have aws config.
 * `--s3-secret-access-key` - /(optional)/ specify S3 secret key
 * `--s3-endpoint` - /(optional)/ specify S3 server URI
 * `--s3-region` - /(optional)/ specify S3 region (default is us-east-1)
+* `--s3-public-read` - /(optional)/ set read-only permission on files uploaded
+  to S3 to an anonymous users
```
Comment (ligurio) [inline README.md:54]: *for* anonymous users?

**Review by ligurio — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (LeonidVas) [reply README.md]: Updated.

**Comment by knazarov** [comment]
LGTM

**Pull request merged**